dask_sql/physical/rex/core/call.py

```diff
@@ -812,11 +853,11 @@ def date_part(self, what, df: SeriesOrScalar):
             return df.dayofyear
         elif what == "HOUR":
             return df.hour
-        elif what == "MICROSECOND":
+        elif what in {"MICROSECOND", "MICROSECONDS"}:
```

Comment:
Nitpicky, but if we add support for singular and plural units, we should probably apply this to all units where it's applicable (minutes, hours, days, etc.); trying to think of if there's a general way we can handle this as part of validating `unit`